test/integration/authorization_test.go

```diff
@@ -1191,7 +1191,7 @@ func TestOldLocalSubjectAccessReviewEndpoint(t *testing.T) {
 			},
 		}
 		actualResponse := &authorizationapi.SubjectAccessReviewResponse{}
-		err := haroldClient.Post().Namespace(namespace).Resource("subjectAccessReviews").Body(sar).Do().Into(actualResponse)
+		err := haroldClient.(*client.Client).Post().Namespace(namespace).Resource("subjectAccessReviews").Body(sar).Do().Into(actualResponse)
```

Comment:
Because I changed the functions to return the client interface instead of the raw client.  I should be able to get rid of the casts by implementing anything missing.